serverless.yml

```diff
@@ -95,6 +96,12 @@ functions:
                   ]
                 ]
 
+  earthdataVarinfo:
+    runtime: python3.9
```

Comment:
Have you tried using 3.11? Lambda supports 3.10 and 3.11 and I _think_ 3.11 should work nicely ([earthdata-varinfo is tested against 3.9, 3.10 and 3.11](https://github.com/nasa/earthdata-varinfo/blob/main/.github/workflows/run_tests.yml#L11). Seems like a good call to use the latest possible option if it works.

Reply:
(Also, it's nice that you can just set the one runtime here, while still having the global option for everything else. Thank you, serverless!)

Reply:
Error:
Configuration error at 'functions.earthdataVarinfo.runtime': must be equal to one of the allowed values [dotnet6, go1.x, java17, java11, java8, java8.al2, nodejs14.x, nodejs16.x, nodejs18.x, provided, provided.al2, python3.7, python3.8, python3.9, python3.10, ruby2.7, ruby3.2]

Reply:
tried 3.10 also. Serverless offline was showing the following: Warning: Warning: found unsupported runtime 'python3.10' for function 'earthdataVarinfo'
✖ Unhandled Lambda Error during invoke of 'graphql-dev-earthdataVarinfo': Error: Unsupported runtime

Reply:
May need to update our serverless version to use the newer runtimes.

Reply:
Yeah - looks that way. Looks like `package.json` has 3.28.1, which was released back in early March, but [Python 3.10 runtime support was merged in mid May](https://github.com/serverless/serverless/pull/11922), and is likely included in 3.31.0.

Reply:
`package.json` just holds the semver, so when you run `npm update serverless` it knows how far its allowed to go. I just ran that command locally and it updated serverless to `3.34.0`. However, it seems its not serverless with the issue, its `serverless-offline`, here is a PR on the repo that awaiting review/merging https://github.com/dherault/serverless-offline/pull/1685 (referenced from https://github.com/dherault/serverless-offline/issues/1716). This would only affect local development though, is there something in 3.10/3.11 that we _need_ or is it just trying to ensure we're running most recent?